ipaplatform/debian/tasks.py

```diff
@@ -8,13 +8,23 @@
 
 from __future__ import absolute_import
 
+import errno
+import logging
+import os
+from pathlib import Path
```

Comment:
IPA doesn't use pathlib yet. Should we start using the stdlib module?

Reply:
If you're happy with Python 3.4 as the minimum?

Reply:
Note that we are not supporting anything below 3.5 already since FreeIPA 4.7.